packages/apps-config/src/ui/icons/accounts.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would really prefer not adding icons and just stick to the fa set. (Happy to change the set, but needs to be consistent)

The issue here is that the base here is not "just for apps", a lot of developers actually fork and add their own custom modules. So doing this has an impact downstream as well.

Additionally, it is easier "picking from a set" than finding correctly licensed svgs.